frontend/packages/console-shared/src/utils/icon-utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as _ from 'lodash';
+import * as operatorLogo from '../images/operator.svg';
+
+export const getImageForCSVIcon = (csv) =>
+  _.get(csv, 'spec.icon')
```

Comment:
Instead of using `get` at three places, probably store it as a `const`.  And to address what Jai said you can do this or something similar
```
const icon =  _.get(csv, 'spec.icon', []);
!_.isEmpty(icon) ?  `data:${icon[0].mediatype};base64,${icon[0].base64data}` : operatorLogo
```

Reply:
Done.